jira/validators/worker.groovy

```diff
@@ -10,6 +10,13 @@ log.info("Start Worker Validation")
 def customFieldManager = ComponentAccessor.getCustomFieldManager()
 def workerCf = customFieldManager.getCustomFieldObjectByName("Worker")
 def worker = issue.getCustomFieldValue(workerCf)
+
+// If resolution is Rejected or Deferred return always true
```

Comment:
I would move this code block to line 8 to avoid loading the CustomFieldManager and save some runtime performance.